### PR TITLE
feat: add documentation

### PR DESCRIPTION
**Pull request by irfan-nawaz**
created landing page and documentation page included typography headings and colors used.

### CODE REVIEW

File: documentation/doc.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+}
+
+.content, .content-row{
+    z-index: -2;
+}
+
+.code-snippet {
+    width: 100%;
+    padding: 1rem 0;
+    background-color: #083D77;
```
Comment (ruchiragoswami) [inline documentation/doc.css:43]: color hard-coded. Suggestion for to use a variable.

File: documentation/doc.css [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    .header-component p{
+        margin: 1rem 3rem;
+    }
+    .nav{
+        width: 11rem;
+    }
+    .code-snippet {
+        width: 70%;
+        padding: 1rem 0;
+        background-color: #083D77;
```
Comment (ruchiragoswami) [inline documentation/doc.css:204]: color hard-coded

**Review by ruchiragoswami — APPROVED** [review]
Looks great Irfan! 
Just a suggestion, I saw you used variables, but at many places, colors are hard-coded too. Is this intentional ? Take a look if you need to change. Approving with comments :)

**Review by Abhijeet43 — APPROVED** [review]
Looks Good!!

**Pull request merged**